CHANGELOG.adoc

```diff
@@ -33,6 +33,7 @@ This will allow us, when we will receive an operation to perform with a given re
 
 - https://github.com/eclipse-sirius/sirius-components/issues/1165[#1165] [doc] Improve the pull request template
 - https://github.com/eclipse-sirius/sirius-components/issues/1155[#1155] [workbench] The left and right panels now use a vertical bar of icons (instead of accordions) to select which view to display
+- https://github.com/eclipse-sirius/sirius-components/issues/1179[#1179] [core] Move `WorkbenchSelection` and `WorkbenchSelectionEntry` to `sirius-components-representations` in order to let any representation use those classes. We may rename the `Selection` representation in the future in order to rename them to `Selection` and `SelectionEntry` to align the frontend and backend API.
```

Comment:
Can it also be considered an API break?

Reply:
It has never been used by anyone outside of sirius-components-emf so I don't consider it as an API but in its new location, it will be used by others so new modifications (if they occur) would be API breaks